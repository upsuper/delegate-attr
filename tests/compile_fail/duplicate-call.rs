use delegate_attr::delegate;

struct Foo(Vec<u8>);

#[delegate(self.0)]
impl Foo {
    #[call(len)]
    #[call(capacity)]
    fn size(&self) -> usize;
}

fn main() {}
