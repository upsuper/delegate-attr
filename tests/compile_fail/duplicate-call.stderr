error: duplicate #[call] attribute
 --> $DIR/duplicate-call.rs:8:5
  |
8 |     #[call(capacity)]
  |     ^
